WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/LargeTitle.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+
+/// Represents a large title, mostly to be used at the top of view controllers like What's New Component
+struct LargeTitle: View {
+    let text: String
+
+    var body: some View {
+        Text(text)
+            .font(.largeTitle)
+            .bold()
```

Comment:
We should probably create a modifier like `.titleStyle()`
Also, doesn't this needs color?

Reply:
Sure thing! I've created a new Style for the LargeTitleStyle 👍 
Done! ✅